test/support/resque/Rakefile

```diff
@@ -11,13 +11,10 @@ require "i18n"
 
 require_relative "../jobs"
 
-redis_url = if ENV["USING_DEV"] == "1"
-  "job-iteration.railgun:6379"
-else
-  "localhost:6379"
-end
+redis_host = ENV.fetch("REDIS_HOST") { "localhost" }
+redis_port = ENV.fetch("REDIS_PORT") { 6379 }
```

Comment:
In what cases would we want the fallback? Also, can't we use `REDIS_URL` directly?